widgets/BigQueryPushdownEngine-sqlengine.json

```diff
@@ -5,7 +5,29 @@
   "display-name": "BigQuery Pushdown",
   "configuration-groups": [
     {
-      "label": "Connection",
```

Comment:
Why are we removing this? The pattern followed in all other GCP plugins is that all the properties coming from `ConnectorConfig` are under `Connection` configuration group.
For eg, https://github.com/data-integrations/google-cloud/blob/develop/widgets/GCSFile-batchsource.json#L8

Reply:
so we had a discussion with @bdmogal on pushdown usability - we don't want to ask the user too much info. As part of the discussion, we want to keep the pushdown widget simple - we have only two configuration groups - `basic` which  will have the `dataset` field that is required and `advanced` has everything else including connection properties. Since this widget is for pushdown broadly and is not specific as other GCP source/sink plugins, I think it is alright to not follow the pattern.

Reply:
Sg, thanks for the info.